### ISSUE TITLE
[PersonSearchElement]: align with searchable dropdown

### ISSUE THREAD

**Issue by odinr** [equinor/fusion-web-components#839]
https://github.com/equinor/fusion-web-components/blob/c9e2a31fcd9b4258931841d5a02742245af7b09d/packages/person/src/components/search/element.ts#L49-L52

`PersonSearchElement` should extend `SearchableDropdownElement`, some minor alignments in `PersonSearchElement` might be needed

**Comment by asbjornhaland** [comment]
@odinr @eikeland Is this issue still relevant after recent updates?

**Comment by eikeland** [comment]
yes, but we need to redefine this and other refactoring requirements to the personSearch controller.
